flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -431,6 +431,26 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
                     .runtimeClass("org.apache.flink.table.runtime.functions.scalar.SplitFunction")
                     .build();
 
+    public static final BuiltInFunctionDefinition URL_DECODE =
+            BuiltInFunctionDefinition.newBuilder()
+                    .name("url_decode")
+                    .kind(SCALAR)
+                    .inputTypeStrategy(sequence(logical(LogicalTypeFamily.CHARACTER_STRING)))
+                    .outputTypeStrategy(nullableIfArgs(explicit(DataTypes.STRING())))
```

Comment:
OK, I will fix.

Reply:
@lincoln-lil IIRC this is the default behavior, I mean `explicit(DataTypes.STRING())` and `explicit(DataTypes.STRING().nullable())` are leading to the same result.
Or did I miss anything?
https://github.com/apache/flink/blob/e8d98f45e2874b7252d757f43ca3b91325cb2c5c/flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/AbstractDataType.java#L46-L54

Reply:
@snuyanzin Sorry I missed the ping here! IIUC the `AbstractDataType`#`nullable` says it's the default behavior means a datatype which doesn't explictly declares its nullabilty then the default behavior is nullable.
For the output type strategy, I think I should point it out more clearly: it should be 'outputTypeStrategy(explicit(DataTypes.STRING().nullable()))' because the return value may be null even if the input is non-null (e.g., decoding failure), WDYT?